### DISCUSSION TITLE
Openroad and QtCharts

### DISCUSSION THREAD

**Discussion by pguerr91** [The-OpenROAD-Project/OpenROAD#4653] [Q&A]
I installed openroad on RHEL8 using the documentation from GitHub. After launching openroad -gui, I noticed that it mentions QtCharts is not installed. So I built and compiled it (from source qtcharts-everywhere-src-5.15.0.zip), and and it appears to have installed in /usr/include/qt5/QtCharts ..but when I launch openroad -gui again , it STILL says "QtCharts not installed" ..(see attachment) Any ideas on what to do ??  This doesn't appear to be very straightforward

![openroad_QtCharts_missing](https://github.com/The-OpenROAD-Project/OpenROAD/assets/47336735/70f5288c-1aa8-447b-ac35-0c0cfb27bee0)

**Comment by maliberty** [comment]
Cmake does some caching so I suggest removing your build directory and re-running cmake to see if it then finds it.

**Reply by pguerr91** [reply]
Sorry, I'm new at this. ..Are you telling me to re-deploy openroad ?  Not sure where to begin..

**Reply by maliberty** [reply]
Are you compiling from source?  I'm suggesting you do a full rebuild (or at least cmake)

**Comment by maliberty** [comment]
Is the rest of Qt installed in the same area?
